deploy.md

```diff
@@ -0,0 +1,70 @@
+# Deploying the API
+
+## Environment
+
+You will need a Linux environment with [Ruby](https://www.ruby-lang.org/en/) installed.  To see which version of Ruby is currently installed, run:
+
+```
+ruby --version
+```
+
+To see which version of Ruby the API currently needs:
+
+```
+cat .ruby-version
+```
+
+If your environment has a different version of Ruby than what the API expects, you can either change the global version or install [rvm](http://rvm.io/) to manage multiple Ruby versions.  Once rvm is installed, you can install the proper version of Ruby by running:
+
+```
+rvm install `cat .ruby-version`
+```
+
+The environment will also need [PostgreSQL](https://www.postgresql.org/).
+
+## The application
+
+The application depends on a handful of Ruby libraries.  These are specified in the Gemfile.  The Gemfile provides a quick way to install all of the libraries at once using `bundler` instead of having to do each one manually.  To install the application library dependencies, run:
+
+```
+# Make sure bundler is installed.
+gem install bundler
+
+# Install from the Gemfile
+bundler install
+```
+
+## Preparing the database
+
```

Comment:
The goal of this installation is that it connects to the oracle reporting database. They are going to need to create a database.yml that has their database specifications. This is where the meat of their effort will be. And it also means they will need to use Capistrano or another script to copy their version over after each code update.